### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by lxbarth** [osmlab/to-fix#69]
Needs docs for 
- Keyboard shortcuts for to-fix
- Loading tasks
- Writing loaders

**Comment by tristen** [comment]
- Keyboard shortcuts for to-fix is complete

**Comment by CloCkWeRX** [comment]
So, loading tasks - where do I want to look to see how that works? Happy to whip up something in markdown or whatever as I blunder through it

**Comment by aaronlidman** [comment]
@CloCkWeRX Loading a new task isn't simple enough that I would recommend it yet. If you want to get your hands dirty though, here's how I did it recently. https://github.com/osmlab/to-fix/commit/c47e7f38c56bac805135e73b82bea9cbfd8817bf

Adding the actual data is done through the modal.
![task](https://cloud.githubusercontent.com/assets/58878/7049542/445a90e2-ddcf-11e4-9c43-dae928eed653.gif)

**Comment by CloCkWeRX** [comment]
Blocked by #5

**Comment by lxbarth** [comment]
Keyboard shortcuts actually only needs documentation in the UI. Creating/adding new tasks on UI and in documentation https://github.com/osmlab/to-fix/issues/132

cc @rub21
